docs/guides/auth.md

```diff
@@ -298,20 +298,31 @@ var result = client.SayHello(request, new CallOptions(credentials: googleCredent
 
 ```php
 // Base case - No encryption/authorization
```

Comment:
Can you switch these to headings rather than just comments in the snippet, as per the other examples?